Jenkinsfile-build

```diff
@@ -0,0 +1,34 @@
+pipeline {
+  agent none
+
+  stages {
+    stage('Test') {
+        agent {
+            ecs {
+                inheritFrom 'ecs'
+            }
+        }
+        steps {
+            checkout scm
+            sh 'sbt dist'
+            stash includes: 'Dockerfile', name: 'Dockerfile'
+            stash includes: 'target/universal/tdr-transfer-frontend-1.0-SNAPSHOT.zip', name: 'tdr-transfer-frontend-1.0-SNAPSHOT.zip'
+        }
+    }
+    stage('Docker') {
+            agent {
+                label 'master'
+            }
+            steps {
+                unstash 'tdr-transfer-frontend-1.0-SNAPSHOT.zip'
+                unstash 'Dockerfile'
+                sh 'docker build -t nationalarchives/tdr-transfer-frontend .'
+                withCredentials([usernamePassword(credentialsId: 'docker', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+                    sh 'echo $PASSWORD | docker login --username $USERNAME --password-stdin'
```

Comment:
It would obfuscate it because it comes from a password variable but this command wouldn't print it anyway, the output of echo is piped to the docker login command.

Reply:
:+1: